modules/ra-gui/src/org/ejbca/ra/EnrollWithRequestIdBean.java

```diff
@@ -621,7 +620,8 @@ private String getFileName() {
         if (StringUtils.isAsciiPrintable(commonName)) {
             return StringTools.stripFilename(commonName);
         }
-        return Base64.encodeBase64String(commonName.getBytes());
+        return  org.apache.commons.lang3.StringUtils.stripAccents(commonName);
```

Comment:
The StringUtils from `org.apache.commons.lang` doesn't have a `stripAccents` method so in order to keep everything else in the code as it was I just went with using the lang3 StringUtils like this